src/Parser/Core/Modules/StatTracker.js

```diff
@@ -203,14 +225,39 @@ class StatTracker extends Analyzer {
       avoidance: this.combatants.selected._combatantInfo.avoidance,
       leech: this.combatants.selected._combatantInfo.leech,
       speed: this.combatants.selected._combatantInfo.speed,
+      armor: this.combatants.selected._combatantInfo.armor,
     };
+
+    this.applySpecModifiers();
+    this.applyArtifactModifiers();
+
     this._currentStats = {
       ...this._pullStats,
     };
 
     debug && this._debugPrintStats(this._currentStats);
   }
 
+  applySpecModifiers() {
+    const modifiers = this.constructor.SPEC_MULTIPLIERS[this.combatants.selected.spec.id];
+    this.applyMultipliers(this._pullStats, modifiers);
+  }
+
+  applyArtifactModifiers() {
+    Object.entries(this.constructor.ARTIFACT_MULTIPLIERS).forEach(([spellId, modifiers]) => {
+      if(this.combatants.selected.traitsBySpellId[spellId] > 0) {
+        this.applyMultipliers(this._pullStats, modifiers);
```

Comment:
Shouldn’t this multiply the value by the number of traits?

Reply:
All the ones I've seen are 0 or 1 but thats a reasonable adjustment.